repo/packages/W/weavescope/0/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packagingVersion": "2.0",
+  "name": "weavescope",
+  "version": "0.14.0",
+  "tags": ["microservices", "visualisation", "monitoring"],
+  "maintainer": "help@weave.works",
+  "description": "**Live visualisation, monitoring and management**\n\nWeave Scope produces a live interactive map of your containerized services and applications with no coding and no instrumentation",
+  "scm": "https://github.com/weaveworks/scope",
+  "website": "http://weave.works/scope",
+  "licenses": [
+    {
+      "name": "Apache License Version 2.0",
+      "url": "https://raw.github.com/weaveworks/scope/master/LICENSE"
+    }
+  ],
+  "postInstallNotes": "Have fun monitoring, visualising and managing you containers with Weave Scope!\n\nYou can access Weave Scope at\n **`https://<dcos_url>/service/weavescope/`**\n\nMore documentation can be found on [Weaveworks website](deploy-scope-on-dcos)."
```

Comment:
based on configuration this documentation is not true.   the end point is:  `https://<dcos_url>/service/weavescope-app/`
if you want `weavescope` you will need to rename the service.name for the app. or this documentation should be updated.